docs/1.getting-started/6.data-fetching.md

```diff
@@ -264,6 +264,60 @@ const { data, error, refresh } = await useFetch('/api/users', {
 })
 ```
 
+Note that **watching a reactive element won't change the URL fetched**. For example, this will keep fetching the same initial ID of the user because the URL is constructed at the moment the function is invoked, even if parameters or a query is passed as an option.
+
+```ts
+const id = ref(1)
+
+const { data, error, refresh } = await useFetch(`/api/users/:user_id`, {
+  params: {
+    user_id: id.value
```

Comment:
this should also work if passing `{ user_id: id }` - we automatically un-ref and make the params reactive if you do this...

Reply:
Does it work with query params also? Last time I checked, query items didn't work.

```ts
const { data, error, refresh } = await useFetch('/api/user/:user_id', {
  user_id: id // Thismay work
  query: {
    with: ['avatar', 'bio'] // No way in hell.
  }
})
```